tests/framework/extensions/clusters/clusters.go

```diff
@@ -18,7 +18,7 @@ import (
 	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
 	"github.com/rancher/rancher/tests/framework/extensions/defaults"
 	"github.com/rancher/rancher/tests/framework/pkg/wait"
-	rancherProvisioning "github.com/rancher/rancher/tests/v2/validation/provisioning"
+	"github.com/rancher/rancher/tests/v2/validation/extensions/provisioning"
```

Comment:
Same point once again, the framework should never import things from v2/validation.

Reply:
oh I misunderstood. I thought you meant that it should not be from the tests folder.